bundles/org.openhab.binding.weathercompany/src/main/java/org/openhab/binding/weathercompany/internal/discovery/WeatherCompanyDiscoveryService.java

```diff
@@ -49,7 +50,6 @@
     private final Logger logger = LoggerFactory.getLogger(WeatherCompanyDiscoveryService.class);
 
     private final LocationProvider locationProvider;
-    private final LocaleProvider localeProvider;
```

Comment:
I might be missing something. But you're deleting the declaration of `localeProvider`, but it's being set in line 65 below? And there's no declaration of `i18nProvider`, but it's also being set below.] in line 66?

Reply:
Yes, because it is already defined in the super class AbstractDiscoveryService.
Like i18nProvider.
It was an error to redefine it.

Reply:
Thanks. Good to know.

I never would've guessed this, but I do see it's in the documentation. 😉 

https://www.openhab.org/docs/developer/bindings/#internationalization-of-discovery-result-labels